package.json

```diff
@@ -49,10 +49,10 @@
     "use-react-version": "node scripts/useReactVersion.mjs",
     "use-material-ui-v6": "node scripts/useMaterialUIv6.mjs",
     "use-material-ui-next": "node scripts/useMaterialUINext.mjs",
-    "build:codesandbox": "pnpm run release:build --concurrency 3",
-    "install:codesandbox": "pnpm install --no-frozen-lockfile",
-    "pkg-pr-new-packages": "node -e \"console.log(Object.values(require('./.codesandbox/ci.json').publishDirectory).join(' '))\"",
-    "pkg-pr-new-release": "pnpm dlx pkg-pr-new publish $(pnpm -s pkg-pr-new-packages) --template './examples/*' --comment=off --peerDeps",
+    "pkg-pr-new:install": "pnpm install --no-frozen-lockfile",
+    "pkg-pr-new:build": "pnpm run release:build",
```

Comment:
Could we just use `release:build` directly if the limited concurrency is no longer needed, as for codesandbox? 🤔